test/release.test.js

```diff
@@ -89,11 +89,19 @@ tap.afterEach(() => {
 })
 
 tap.test('Should delete the release if the pr is not merged', async t => {
-  const { release } = setup()
+  const { release, stubs } = setup()
   const data = clone(DEFAULT_ACTION_DATA)
   data.context.payload.pull_request.merged = false
   await release(data)
 
+  t.ok(
+    stubs.runSpawnStub.calledWith('git', [
```

Comment:
I realize you may have found this pattern in the code already, but these assertions are better done with https://sinonjs.org/releases/latest/assertions/. if you do them with tap's t.ok, when the assertion fails you won't get any details about why it failed but just a generic error message

Reply:
That's good to know, it would have made writing the tests much easier! I've updated the rest of the tests.